docs/ad/index.md

```diff
@@ -40,36 +45,54 @@ A feature is the field in your index that you want to find anomalies for. A dete
-4. Choose a **Field**.
-5. Choose **Save**.
+1. On the **Features** page, select **Add features**.
+1. Enter the **Name** of the feature.
+1. For **Find anomalies based on**, choose the method to find anomalies. For **Field Value** menu, choose the **field** and the **aggregation method**. Or choose **Custom expression**, and add in your own JSON aggregation query.
+1. Preview sample anomalies and adjust the feature settings if needed.
+- For sample previews, the anomaly detection plugin selects a small number of data samples, for example, one data point for every 30 minutes, and uses interpolation to estimate the remaining data points to approximates the actual feature data. It loads this sample dataset into the detector. The detector uses this sample dataset to generate a sample preview of anomaly results.
+Examine the sample preview and use it to fine-tune your feature configurations, for example, enable or disable features, to get more accurate results.
+1. Choose **Save**.
+1. Choose between automatically starting the detector (recommended) or manually starting the detector at a later time.
 
 ### Step 3: Observe the results
 
-The **Total anomalies** graph plots the anomaly grade with the corresponding measure of confidence.
+Choose the **Anomaly results** tab.
+
+You will have to wait for some time to see the anomaly results.
+The detector needs at least 6 data points in 8 continuous intervals (shingles) to initialize.
+So, if the detector interval is 10 min, the detector might take more than an hour to initialize.
+If you see the detector pending in initialization for a long time, increase the detector interval to fasten the process.
+Or, use the [profile detector](./api#profile-detector) operation to make sure you have sufficient data in a shingle size of 8.
+
+![Anomaly detection results](../images/ad.png)
+
+The **Live anomalies** chart shows you the live anomaly results for the last 60 intervals. For example, if the interval is set to 10, it shows the results for the last 600 minutes.
+
+The **Anomaly history** chart plots the anomaly grade with the corresponding measure of confidence.
 
 Anomaly grade is a number between 0 and 1 that indicates the level of severity of how anomalous a data point is. An anomaly grade of 0 represents “not an anomaly,” and a non-zero value represents the relative severity of the anomaly. The confidence score is an estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
 
-The **Features** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
+The **Feature breakdown** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
```

Comment:
Should we describe the "Anomaly occurrence" table?

Reply:
Added.